Library/Formula/node.rb

```diff
@@ -14,6 +14,7 @@ class Node < Formula
   end
 
   option "with-debug", "Build with debugger hooks"
+  option "with-icu4c", "Build with Intl (icu4c) support"
```

Comment:
> It reflects this choice when configuring node:
https://github.com/nodejs/node#intl-ecma-402-support

I see. I like that there are a few different options now.

> I ported what we were doing for iojs here: https://github.com/aredridel/homebrew-iojs/blob/master/Formula/iojs.rb

So if I understanding correctly, the "default" is becoming ` --with-intl=system-icu`, (a decision which I heartily concur with; I think its the best option), and the homebrew install flag '`-with-icu4c` is to force the build to link with homebrew's 'icu4c' package? May I suggest then that the option be changed to `--with-brewed-icu4c`, a la the `git.rb` formula's options for openssl, ssl, and curl?